src/analysis/retail/evoker/devastation/modules/talents/SpellweaversDominance.tsx

```diff
@@ -0,0 +1,73 @@
+import SPELLS from 'common/SPELLS';
+import TALENTS from 'common/TALENTS/evoker';
+import { formatNumber } from 'common/format';
+import HIT_TYPES from 'game/HIT_TYPES';
+
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import ItemDamageDone from 'parser/ui/ItemDamageDone';
+import Events, { DamageEvent } from 'parser/core/Events';
+
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import { SPELLWEAVERS_DOMINANCE_CRIT_MULTIPLIER } from 'analysis/retail/evoker/devastation/constants';
+import TalentSpellText from 'parser/ui/TalentSpellText';
+
+class SpellweaversDominance extends Analyzer {
+  critDamage: number = 0;
+  SpellweaversDominanceDamage: number = 0;
+  heatWaveDamageCritMultiplier: number = SPELLWEAVERS_DOMINANCE_CRIT_MULTIPLIER;
+  spellsToTrack = [
+    SPELLS.DISINTEGRATE,
+    SPELLS.FIRE_BREATH_DOT,
+    SPELLS.ETERNITY_SURGE_DAM,
+    SPELLS.LIVING_FLAME_DAMAGE,
+    SPELLS.PYRE,
+    SPELLS.OBSIDIAN_SHARDS,
```

Comment:
is not present in the whitelist apparently

Reply:
Fixed. Ended up not mattering, interestingly the DoT doesn't appear to be critting at all.